src/lib/market-service/portals/portals.ts

```diff
@@ -115,11 +113,7 @@ export class PortalsMarketService implements MarketService {
     }
   }
   async findByAssetId({ assetId }: MarketDataArgs): Promise<MarketData | null> {
-    const assets = generatedAssetData as unknown as AssetsByIdPartial
-
     try {
-      const asset = assets[assetId]
-      if (!asset) return null
```

Comment:
That was a big rug - this could never work with runtime assets, since these do *not* exist in generatedAssetData.json by definition